### ISSUE TITLE
K/DNX installation location changed

### ISSUE THREAD

**Issue by Eilon** [aspnet/Announcements#5]
The core runtime, tools, and NuGet packages are installed into a sub-folder in `%USERPROFILE%`. Previous names of these folders were `.k`, `.kpm`, and `.kre`.

The new folders are all structured under a top-level `.dnx` folder, but otherwise have very similar contents to what they used to have.

```
%USERPROFILE%\.dnx\alias\default.txt
%USERPROFILE%\.dnx\packages\<NuGet packages>
%USERPROFILE%\.dnx\runtimes\dnx-clr-win-x64.1.0.0-beta4\...
%USERPROFILE%\.dnx\runtimes\dnx-clr-win-x86.1.0.0-beta4\...
%USERPROFILE%\.dnx\runtimes\dnx-coreclr-win-x64.1.0.0-beta4\...
%USERPROFILE%\.dnx\runtimes\dnx-coreclr-win-x86.1.0.0-beta4\...
```

**Comment by CumpsD** [comment]
Any way to override this? I run a mobile drive setup with everything in a different path.

**Comment by alexsandro-xpt** [comment]
Tip: May the root path should be parameterized, to change in another directory for advanced users, not everything in %USERPROFILE%

**Comment by MichaCo** [comment]
Yup, it would be great to be able to control the `.dnx` folder location. User profile for me is on the C drive, which is 90% full on my box for example ;)
The `packages` folder can get big! 2GB+ easily....

**Comment by TheMadKow** [comment]
Agreed. To people with rather small SSD drive, this may become an issue real fast.

**Comment by aggieben** [comment]
This can already be accomplished with the `DNX_USER_HOME` environment variable.  You can set it up like this:

```
curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_USER_HOME=$DNX_USER_HOME sh
```

**Comment by CumpsD** [comment]
@aggieben that's a `.sh`, we're talking plain Windows

**Comment by jods4** [comment]
To people who have small SSD for their main (C:) drive, a possible solution is to create a NTFS junction at `%USERPROFILE%\.dnx` and move it to another disk. I did this to free some space of the windows update folder <_<

**Comment by CumpsD** [comment]
@jods4 those are just patches, combatting symptoms instead of causes. I'm sure out of all developers, you'll find some who don't want to touch `%USERPROFILE%` at all for some reason :)

The `DNX_USER_HOME` seems to exist in the `.ps1` install files too, I don't know if it is documented somewhere?

**Comment by akoeplinger** [comment]
Setting `DNX_HOME` works fine:

![image](https://cloud.githubusercontent.com/assets/1376924/8936184/2228f8fe-3551-11e5-935a-725413ba8b22.png)

**Comment by CumpsD** [comment]
@akoeplinger thanks for confirming!

**Comment by alexsandro-xpt** [comment]
My tip really is just about SSD drive, lot of things alredy in %USERPROFILE% and create a NTFS junction at `%USERPROFILE%\.dnx` to another disk is boring to manage.

**Comment by MichaCo** [comment]
@akoeplinger If I change the dnx_home and remove the installed DNX runtime versions from the user profile, VS 2015 stops working. At least for me, it doesn't find any DNX anymore and cannot compile the solutions. DNX.targets throws an error `The Dnx Runtime package needs to be installed`...

If I setup DNVM again with `DNVM setup` and install some runtimes, it starts working again. So it seams VS doesn't care about the path or DNX_HOME env variable -.-
